src/components/ContentCard/ContentCard.component.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { CardContent, CardActionArea } from '@material-ui/core';
+import { useHistory } from 'react-router-dom';
+
+import { StyledCard, StyledCardMedia, Title, Description } from './ContentCard.styles';
+
+const ContentCard = ({ item }) => {
+  const history = useHistory();
+
+  const handleOnClick = () => history.push(`/v/${item.id.videoId}`);
+
+  return (
+    <StyledCard onClick={handleOnClick}>
+      <CardActionArea>
+        <StyledCardMedia
+          src={item.snippet.thumbnails.high.url}
+          title={item.snippet.title}
+        />
+        <CardContent>
+          <Title dangerouslySetInnerHTML={{ __html: item.snippet.title }} />
```

Comment:
Is this `dangerouslySetInnerHTML` really necessary? I think you could do something like this instead:

```suggestion
          <Title>{ item.snippet.title}</Title>
```

Reply:
I mostly use it for the elements to interpret HTML codes from string values